packages/fxa-auth-server/fxa-oauth-server/lib/routes/introspect.js

```diff
@@ -66,18 +66,25 @@ module.exports = {
 
     if (token) {
       if (token.expiresAt) {
-        response.active = (+token.expiresAt < Date.now());
+        response.active = (+token.expiresAt > Date.now());
```

Comment:
Cool the `+token.expiresAt` actually converts the Date object to ms since epoch 👍🏽